robottelo/datafactory.py

```diff
@@ -388,6 +425,12 @@ def valid_http_credentials(url_encoded=False):
             u'quote': False,
             u'http_valid': False,
         },
+        {
+            u'login': gen_string('utf8', 10),
```

Comment:
If I'm not wrong gen_string second parameter is 10 for default, we can omit it here.